### PR TITLE
Add documentation on model operations

### PR DESCRIPTION
**Pull request by planger**
Contributed on behalf of STMicroelectronics.

### CODE REVIEW

**Comment by planger** [comment]
Preview: https://deploy-preview-78--glsp.netlify.app/documentation/modeloperations/

File: content/documentation/modelOperations/content.md [hunk 1]
```diff
@@ -8,4 +8,127 @@ title = "Model Operations"
...
+        return {
+            kind: KIND,
+            isOperation: true,
+            elementId,
+            ...options
+        };
+    }
+}
+```
+
```
Comment (tortmayr) [inline content/documentation/modelOperations/content.md:44]: Maybe we should also mention here that most of the default operation types  have a generic args map? So minor customization (e.g. passing additional context element ids) can be done without having to create a new custom operation type.

File: content/documentation/modelOperations/content.md [hunk 2]
```diff
@@ -8,4 +8,127 @@ title = "Model Operations"
...
+See also [the process of editing the source model]({{< ref "sourceModel" >}}#processing-edit-operations).
+
+Also, the operation action handler is a good place in general to introduce any other steps that need to be performed before or after each model manipulation, such as triggering live validation, sending notifications to external services, etc.
+Thus, it isn't unusual to overwrite the operation action handler for specific scenarios to inject before/after modification steps.
+
+The operation action handler is typically agnostic of the actual source model language; it usually is, however, specific to the source model framework (e.g. EMF, JSON, etc.) as it needs to encapsulate the modification in framework-specific command API to support native undo/redo, or take snapshots before and after the manipulation and compute a patch, etc.
+Thus, for each [source model framework integration]({{< ref "integrations" >}}#source-model-integrations), there typically is a dedicated operation action handler, which often also entails a specific interface for the operation handler implementations.
```
Comment (tortmayr) [inline content/documentation/modelOperations/content.md:132]: Should we mention here that we provide base libraries with reusable operation handlers for the emf/gmodel-only/json uses cases?

**Review by tortmayr — APPROVED** [review]
Thanks Philip, nice work!
The new documentation looks good to me.
I only have a few optional suggestions but we can also merge this as is.

**Comment by planger** [comment]
@tortmayr Thank you very much for your fast and helpful feedback! I've added the Java code example and added a few lines to address your comments (hopefully) :-)

**Pull request merged**